### PR TITLE
Enhancement/hp phenotype files

### PR DESCRIPTION
**Pull request by iimpulse**

### CODE REVIEW

File: src/ontology/hp.Makefile [hunk 1]
```diff
@@ -369,19 +369,13 @@ hpoa:
...
 		&& $(MAKE) -C current
-	cd $(RARE_DISEASE_DIR)/util/ \
-		&& echo "##### HPOA: Running ANNOTATION Makefile (UTIL)" \
-		&& $(MAKE) -C annotation
 	
 	echo "##### HPOA: COPYING all result files into HPOA results directory"
 	mkdir -p $(HPOA_DIR)
-	cp $(RARE_DISEASE_DIR)/util/annotation/genes_to_phenotype.txt $(HPOA_DIR)
-	cp $(RARE_DISEASE_DIR)/util/annotation/phenotype_to_genes.txt $(HPOA_DIR)
-	cp $(RARE_DISEASE_DIR)/misc/*.tab $(HPOA_DIR)
```
Comment (matentzn) [inline src/ontology/hp.Makefile:384]: We have been releasing

[phenotype_annotation.tab](https://github.com/obophenotype/human-phenotype-ontology/releases/download/v2023-01-27/phenotype_annotation.tab)

and

[phenotype_annotation_negated.tab](https://github.com/obophenotype/human-phenotype-ontology/releases/download/v2023-01-27/phenotype_annotation_negated.tab)

What about these files? Are they deprecated from now on?

Comment (iimpulse) [reply src/ontology/hp.Makefile:384]: Yes these are no longer in the pipeline and have't been used for awhile. Some external folks may still use them but these were deprecated in 2021 in favor of phenotype.hpoa.

Also, with the recent merge of monarch-initiative/hpo-annotation-data#876 misc and util directories no longer exist.

**Comment by matentzn** [comment]
> We are thinking of moving forward with apache parquet releases for speed etc (probably in addition to plain tsv) so it would be good to have just one format.

Definitely keep a TSV file as well :D

File: src/ontology/hp.Makefile (hunk 1 above)
Comment (matentzn) [reply src/ontology/hp.Makefile:384]: Yeah I know, I am just saying that we remove a release file, but it makes sense if it was already depreacted.

**Comment by drseb** [comment]
Please only deprecate the old pipelines and not deprecate the files. Many users are dependant on the tsv format.

**Comment by pnrobinson** [comment]
we have been saying we would deprecate these files for 3 years now -- do we have some sense of how big an issue it would be for users? Should we ask on the HPO mailing list for example?

**Comment by drseb** [comment]
No i mean the other tsv files. Do you want to keep those?

**Comment by drseb** [comment]
The ones called hpoa etc. 

Are you planning to remove all gene-phenotype files?

**Comment by matentzn** [comment]
100%, do not deprecate them. I dont really like this parquet idea - our files are way too small to make this worthwhile, and know one I know knows how to parse these.. Not saying its not useful for some cases, but this is small data.

**Comment by pnrobinson** [comment]
Suggestion -- keep all of the files currently on https://hpo.jax.org/app/data/annotations
add a parquet-formated phenotype.hpoa
also add the various MAXO release files here.
does this sound OK?

**Comment by matentzn** [comment]
Sounds great!

**Comment by iimpulse** [comment]
> The ones called hpoa etc.
> 
> Are you planning to remove all gene-phenotype files?

The output files would be:

All regular ontology files  --- untouched
phenotype.hpoa -- untouched
phenotype_to_genes.txt -- new from phenol in sync with hpo-web.. file format changed - breaking
genes_to_phenotypes.txt -- new from phenol in sync with hpo-web.. file format changed - breaking

Future Files not in this change
maxo files
phenotype.hpoa.parquet

**Comment by drseb** [comment]
Sounds good!

Am Do., 16. März 2023 um 22:57 Uhr schrieb iimpulse <
***@***.***>:

**Comment by iimpulse** [comment]
hi @matentzn, the rest of the systems are in place for this to be merge. You can test a release locally to see that the files are the same as the main branch (but obviously updated)

**Comment by matentzn** [comment]
Forget what I just said, ignore!

**Comment by matentzn** [comment]
Works great

**Pull request merged**